resip/recon/ConversationManager.cxx

```diff
@@ -1223,6 +1226,30 @@ ConversationManager::onTryingNextTarget(AppDialogSetHandle, const SipMessage& ms
    return true;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+// Pager Handler ///////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////////////////////////
+
+const char*
+ConversationManager::sendMessage(const NameAddr& destination, const Data& msg, const Mime& mimeType)
+{
+   if(!mUserAgent->mDum.getMasterProfile()->isMethodSupported(MESSAGE))
+   {
+      mUserAgent->mDum.getMasterProfile()->addSupportedMethod(MESSAGE);
+   }
+   if(!mUserAgent->mDum.getMasterProfile()->isMimeTypeSupported(MESSAGE, mimeType))
+   {
+      mUserAgent->mDum.getMasterProfile()->addSupportedMimeType(MESSAGE, mimeType);
+   }
+   
+   ClientPagerMessageHandle cpmh = mUserAgent->mDum.makePagerMessage(destination);
+   auto_ptr<Contents> msgContent(new PlainContents(msg));
```

Comment:
please use the PlainContents constructor that also takes the MIME type as a second argument